cpp/include/rmm/mr/pinned_host_memory_resource.hpp

```diff
@@ -17,7 +17,7 @@
 
 #include <rmm/aligned.hpp>
 #include <rmm/detail/aligned.hpp>
-#include <rmm/detail/cccl_adapters.hpp>
+#include <rmm/detail/cccl_adaptors.hpp>
```

Comment:
Sorry @bdice: what is the difference between adapter vs adaptor?

Reply:
This is a British / American spelling difference. We actually use the British form in RMM, and never the American form. I am changing this just for consistency with RMM's existing precedent.